htdocs/admin/pdf.php

```diff
@@ -291,16 +291,16 @@
 print '</td></tr>';
 
 print '<tr class="oddeven"><td>'.$langs->trans("MAIN_PDF_MARGIN_LEFT").'</td><td>';
-print '<input type="text" class="maxwidth50" name="MAIN_PDF_MARGIN_LEFT" value="'.(empty($conf->global->MAIN_PDF_MARGIN_LEFT) ? 10 : $conf->global->MAIN_PDF_MARGIN_LEFT).'">';
```

Comment:
@grandoc you can give the default value in second parameter so you can replace
`(empty($conf->global->MAIN_PDF_MARGIN_LEFT) ? 10 : $conf->global->MAIN_PDF_MARGIN_LEFT)`
by 
`getDolGlobalInt('MAIN_PDF_MARGIN_LEFT', 10)`